src/main/java/frc/robot/subsystems/tiltedelevator/TiltedElevatorSubsystem.java

```diff
@@ -23,21 +22,45 @@
 import frc.robot.util.MotorUtil;
 import frc.robot.util.ShuffleboardUtil;
 import frc.robot.motorcontrol.HallEffectSensor;
+import frc.robot.subsystems.tiltedelevator.ElevatorState.OffsetState;
 import frc.robot.motorcontrol.HallEffectMagnet;
 
 import static frc.robot.Constants.TiltedElevatorConstants.*;
 
 public class TiltedElevatorSubsystem extends SubsystemBase {
+    // config
```

Comment:
Is config desired above the motors?

Reply:
yes, bc i don't think we'll need to look at / change the motors very often